src/test/java/bozo/TaskListTest.java

```diff
@@ -0,0 +1,31 @@
+package bozo;
+
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+public class TaskListTest {
+    @Test
+    public void testAddTask() {
```

Comment:
I like how thorough you are with test cases! However I would like to know whether the tests are testing for success or failure, this can be shown by altering the name to be like Todo_addTask_success() or Todo_removeTask_success() so we know that we are testing for success cases and not fail cases.